nrich-core-api/src/main/java/net/croz/nrich/core/api/exception/ExceptionWithMessage.java

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright 2020-2023 CROZ d.o.o, the original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package net.croz.nrich.core.api.exception;
+
+/**
+ * Marker interface that uses exceptions message for notification.
+ */
+public interface ExceptionWithMessage {
+
+    String getMessage();
```

Comment:
You can remove this method and use the interface as a pure marker. There is already a `getMessage` method for exceptions.